crates/sui-indexer/README.md

```diff
@@ -13,7 +13,7 @@ export PATH="/opt/homebrew/opt/postgresql@15/bin:$PATH"
 ```
 - make sure you have libpq installed: `brew install libpq`, and in your profile, add `export PATH="/opt/homebrew/opt/libpq/bin:$PATH"`. If this doesn't work, try `brew link --force libpq`.
 
-- install Diesel CLI with `cargo install diesel_cli --no-default-features --features postgres`, refer to [Diesel Getting Started guide](https://diesel.rs/guides/getting-started) for more details
+- install Diesel CLI with `cargo install diesel_cli --no-default-features --features indexer`, refer to [Diesel Getting Started guide](https://diesel.rs/guides/getting-started) for more details
```

Comment:
This one shouldn't change -- this is the feature for diesel itself, not our own feature.

Reply:
Ah yes, did a replace in all files :(